packages/vm/Cargo.toml

```diff
@@ -7,6 +7,18 @@ description = "VM bindings to run cosmwams contracts"
 repository = "https://github.com/CosmWasm/cosmwasm/tree/main/packages/vm"
 license = "Apache-2.0"
 
+[package.metadata.release]
+pre-release-hook = [
+    "../../devtools/release_checks.sh"
+]
+pre-release-replacements = [
+    { file="../../CHANGELOG.md", search="## \\[Unreleased\\]", replace="## [{{version}}] - {{date}}", exactly=1 },
+    { file="../../CHANGELOG.md", search="Unreleased", replace="{{version}}" },
```

Comment:
Looks like prettier does not like capitalized `[Unreleased]:`.
Also, can we use a more specific regex (like "\\[unreleased\\]:")? I want to avoid accidentally replacing "unreleased" in the changelog text.